recipes/gneiss/meta.yaml

```diff
@@ -0,0 +1,33 @@
+package:
+  name: gneiss
+  version: 0.1.0
+
+source:
+  git_url: https://github.com/biocore/gneiss.git
+
+
+requirements:
+  build:
+    - python
+    - scikit-bio ==0.4.2
+    - pandas
+    - scipy
+    - numpy
+    - matplotlib
+  run:
+    - python
+    - scikit-bio ==0.4.2
+    - pandas
+    - scipy
+    - numpy
+    - matplotlib
+
+test:
+  # Python imports
+  imports:
+    - gneiss
+
+about:
+  home: https://github.com/biocore/gneiss
+  license: GPLv3 License
```

Comment:
Just a note for other reviewers, we need to be GPLv3 for the moment due to imports from ete3. However, we plan to switch to BSD as soon as possible, assuming the ete3 development team is interested and able to dual license.